### ISSUE TITLE
[BUG] win_service module throwing unneccesary error

### ISSUE THREAD

**Issue by poliscialex** [saltstack/salt#63577]
### Description of Issue
When running a service.status using win_service, if the service is not installed, rather than returning `False`, Salt returns `RROR: Failed To Open SERVICENAME: The specified service does not exist as an installed service.` This also has an impact on beacons, because if a beacon is monitoring a service that is not installed on a particular minion, it fills the event bus up with this:

```
salt/beacon/minion1/service/        {
    "_stamp": "2023-01-27T00:45:47.437422"
}
```

### Steps to Reproduce Issue
Run `service.status` against a windows minion for a service that is not installed

### Versions Report
```
Salt Version:
          Salt: 3005.1
 
Dependency Versions:
          cffi: 1.14.6
      cherrypy: unknown
      dateutil: 2.8.1
     docker-py: 6.0.1
         gitdb: Not Installed
     gitpython: Not Installed
        Jinja2: 3.1.0
       libgit2: Not Installed
      M2Crypto: Not Installed
          Mako: Not Installed
       msgpack: 1.0.2
  msgpack-pure: Not Installed
  mysql-python: Not Installed
     pycparser: 2.21
      pycrypto: Not Installed
  pycryptodome: 3.9.8
        pygit2: Not Installed
        Python: 3.9.16 (main, Jan  6 2023, 22:49:56)
  python-gnupg: 0.4.8
        PyYAML: 5.4.1
         PyZMQ: 23.2.0
         smmap: Not Installed
       timelib: 0.2.4
       Tornado: 4.5.3
           ZMQ: 4.3.4
 
System Versions:
          dist: ubuntu 20.04 focal
        locale: utf-8
       machine: x86_64
       release: 5.4.0-137-generic
        system: Linux
       version: Ubuntu 20.04 focal
  ```

**Comment by twangboy** [comment]
https://saltstackcommunity.slack.com/archives/C7U9FLK8S/p1674776137683189

**Comment by twangboy** [comment]
@whytewolf
  [6 hours ago](https://saltstackcommunity.slack.com/archives/C7U9FLK8S/p1674843307892779?thread_ts=1674776137.683189&cid=C7U9FLK8S)
ok, found what is going on. it is a problem is with win_service throwing an exception instead of just saying False when the service doesn't exist. There needs to be a bug filed for this. the good is this only effects windows systems. the bad is that win_service should never have broken that bad. modules should not throw exceptions expect when something goes horribly wrong. a package not being installed is not something that is horribly wrong.

**Comment by twangboy** [comment]
I belive this has been fixed by #64001
